docs/components/links/README.md

```diff
@@ -277,26 +315,24 @@ Display icon on right of link message.
 
 ## Installation
 
-Resources are available within the [cdr-link package](https://www.npmjs.com/package/@rei/cdr-link):
-
-<cdr-doc-api type="installation" />
+Resources are available within the [cdrLink package](https://www.npmjs.com/package/@rei/cdr-link):
 
 - Component: `@rei/cdr-link`
 - Component styles: `cdr-link.css`
 
 To incorporate the required assets for a component, use the following steps:
 
-### #1. Install using NPM
+### 1. Install using NPM
 
-Install the `cdr-link` package using **npm** in your terminal:
+Install the `cdrLink` package using `npm` in your terminal:
```

Comment:
According to the doc, `cdrLink` should be CdrLink and should not have code formatting